runtime/queries/poe_filter/highlights.scm

```diff
@@ -36,6 +36,6 @@
 "\"" @punctuation.delimiter
 
 ; conceal unnecessary quotes
-("\"" @conceal
-  (#not-has-parent? @conceal string file)
+("\"" @string
+  (#not-has-parent? @string string file)
```

Comment:
```suggestion
("\"" @punctuation.delimiter
  (#not-has-parent? @punctuation.delimiter string file)
```
Would a `@_conceal` capture work for conceal-only queries?

Reply:
I'd prefer `@conceal`, as `@_conceal` will still be applied

Reply:
But we'd have to document `@conceal` for it to work.

Reply:
So will `@conceal`? There's nothing special about that in the treesitter highlighter.

Reply:
And the point of these non-conceal captures is to have the expected highlighting if `conceal` is not set.

Reply:
> But we'd have to document `@conceal` for it to work.

Do we need to? I thought @clason has already mentioned conceals in `CONTRIBUTING.md`

Reply:
I have documented that you do _not_ need `@conceal`.

Reply:
> And the point of these non-conceal captures is to have the expected highlighting if `conceallevel` is not set.

But they're already captured before.

Reply:
> And the point of these non-conceal captures is to have the expected highlighting if `conceallevel` is not set.

But they're already highlighted.

Reply:
Then the patterns should be combined.

Reply:
No because they shouldn't always be concealed.